tests/test_manip_tools.py

```diff
@@ -154,13 +155,14 @@ def test_stringclade(self):
     def test_taxtree(self):
         taxdict = mt.TaxDict(idents=idents, ranks=ranks, lineages=lineages)
         treestring = mt.taxTree(taxdict)
+        print(treestring)
         # not the best test as newick string order can be arbritrary
-        self.assertEqual(treestring, '(((((((Homo_sapiens:4.0,Gorilla_gorilla:\
-4.0)Homininae_subfamily:1.0,Pongo_pongo:3.0)Hominidae_family:2.0,Macca_mulatta:\
-7.0)Catarrhini_parvorder:4.0,Mus_musculus:11.0)Euarchontoglires_superorder:4.0,\
-(Ailuropoda_melanoleuca:9.0,Ailurus_fulgens:9.0)Caniformia_suborder:6.0,\
-Chlorotalpa_tytonis:13.0)Mammalia_class:5.0,Arabidopsis_thaliana:20.0)\
-Eukaryota_superkingdom:1.0,Bacillus_subtilus:21.0)life;')
+        self.assertEqual(treestring, '((((Ailuropoda_melanoleuca:9.0,\
```

Comment:
In the previous version, running taxTree in Python3 resulted in trees for which the ordering of branches was random, causing this test to (usually) fail. This tree is topologically identical to the previous test, but ordered appropriately for the new output.